src/is_face_detector/faceDetection.py

```diff
@@ -6,17 +6,14 @@
 
 
 class FaceDetector:
```

Comment:
create a pattern to file names, try to avoid mix `camelCase` and `snake-case`. I would recommend to use `snake-case`.